notes/deploy.sh

```diff
@@ -73,4 +63,16 @@ git config user.name "Travis CI"
 git config user.email "$AUTHOR_EMAIL"
 git diff-index --quiet HEAD || git commit -m 'Updating Documentation'
 git checkout -b "$DEPLOY_BRANCH"
+
+# skip deploy if we are not on our master branch
```

Comment:
I believe we could (and should) move this above all the git commands (before the `log "going to deploy to"`)